ext/opcache/ZendAccelerator.c

```diff
@@ -3300,6 +3302,52 @@ static zend_result accel_post_startup(void)
 #endif
 		accel_shared_globals = calloc(1, sizeof(zend_accel_shared_globals));
 	}
+
+	/* opcache.file_cache_read_only should only be enabled when all script files are read-only */
+	int file_cache_access_mode = 0;
+
+	if (ZCG(accel_directives).file_cache_read_only) {
+		zend_accel_error_noreturn(ACCEL_LOG_INFO, "opcache.file_cache is in read-only mode");
```

Comment:
This doesn't work, it bombs out after logging it, rather than log & continue as expected.